custom_op/user_ops/kernels/matrix_add_kernel_gpu.cu.cc

```diff
@@ -5,31 +5,44 @@
 #define EIGEN_USE_GPU
 
 #include "matrix_add_op.h"
```

Comment:
These changes are from the `tf1.14` branch. Please drop these commits.
This could be done by

```
git checkout master
git pull origin master -ff
git checkout <your-branch>
git rebase -i master
```

and instead of `pick` use `drop`.